extensions/extensions-helm/aissemble-kafka-chart/values.yaml

```diff
@@ -19,9 +14,15 @@ kafka:
   interBrokerListenerName: CLIENT
   # Listener Ports Configuration
   listeners:
-    - "CLIENT://0.0.0.0:9093"
-    - "EXTERNAL://0.0.0.0:9092"
-    - "CONTROLLER://0.0.0.0:9097"
+    ## @param listeners.client.name Name for the Kafka client listener
```

Comment:
As part of the Kafka Helm chart upgrade, the configuration of the listeners have changed. This is to accommodate for that change.

Reply:
Is this info in the docs for the base chart? If so we can leave it out of our file.